core/base/inc/ROOT/RConfig.hxx

```diff
@@ -517,6 +496,15 @@
 # define _R__DEPRECATED_622(REASON) _R_DEPRECATED_REMOVE_NOW(REASON)
 #endif
 
+/* To be removed by 6.24 */
+#if ROOT_VERSION_CODE <= ROOT_VERSION(6,20,0)
```

Comment:
Since the master will (very soon) be at 6.21, it sounds like it will never trigger ?

Reply:
True, I removed that.